internal/core/command/device_test.go

```diff
@@ -17,10 +31,21 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
 )
 
-//commandByDeviceID
-func TestExecuteGETCommandByDeviceIdAndCommandId(t *testing.T) {
+var (
+	status400 = "400"
+	status404 = "404"
+	status500 = "500"
+	status423 = "423"
+
+	mismatch = "d200-c200-mismatch"
+	d200c404 = "d200-c404"
+	d200c500 = "d200-c500"
+
+	TestCommandId = "TestCommandID"
+)
 
-	LoggingClient = logger.NewMockClient()
+// commandByDeviceID
+func TestExecuteGETCommandByDeviceIdAndCommandId(t *testing.T) {
 	mdc = newMockDeviceClient()
 	dbClient = newCommandMock()
```

Comment:
Commented here; potentially applies elsewhere.

I think these should be moved inside the test loop.

This ensures fresh instances for each test (and avoids the potential for cross-test contamination).

Reply:
These are setting the global variables that are getting removed anyway in my upcoming work. Refactoring it without removing them seems unnecessary and an entry point for bugs, but if you insist I can do it.

Reply:
Makes sense.  I was reviewing within the context of the issue and didn't take into account the follow-on issues related to this.